src/main/java/server/Server.java

```diff
@@ -0,0 +1,40 @@
+package main.java.server;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.logging.Level;
+
+class Server {
+    private final Integer port;
+    private boolean serverIsRunning = true;
+
+    private Server(Integer port) {
+        this.port = port;
+    }
+
+    public static void main(String[] args) throws IOException {
+            int port = Integer.parseInt(args[0]);
```

Comment:
What happens if I don't pass any args to this function?

Also, maybe we should create a story for wrapping the arg in a CLI flag (e.g. something you'd set with `--port=5000`). That's usually a good call for CLI programs so that you can respond to flags like `--help` and support an arbitrary number of args without major cognitive overhead for the user (e.g. "does this argument go seventh or eighth?").